src/lua/datetime.lua

```diff
@@ -1114,12 +1114,32 @@ ffi.metatype(datetime_t, {
     __index = datetime_index,
```

Comment:
I believe there was a doc-request about field of INTERVAL: in this [PR](https://github.com/tarantool/tarantool/pull/6923) we have commit [datetime: interval changelog and doc request](https://github.com/tsafin/tarantool/commit/669204071aa0a4219d87e136246b5d12b0722031).

Reply:
But the doc request doesn't mention that the interval is an object which has these fields. The fields are described as arguments for sub/add/new. It means that if in the future we will decide to store hours mins and secs, for example, all as secs, how will `totable()` work then?
I don't mind keeping it but addition of a new public method just for internal testing I would rather recommend to avoid. Otherwise you will need to support it forever regardless of how the interval struct will mutate.